packages/11ty/content/_computed/eleventyComputed.js

```diff
@@ -149,42 +209,20 @@ export default {
    */
   publicationContributors: ({ collections, config, page, publication }) => {
     if (!collections.all) return
-    let publicationContributors = Array.isArray(publication.contributor)
-      ? publication.contributor
-      : []
-    publicationContributors = publicationContributors.filter((item) => item)
-    if (!publicationContributors.length) return
+    if (!publication.contributor) return []
 
-    /**
-     * Add `pages` properties to contributor with limited `page` model
-     */
-    const addPages = (contributor) => {
-      const { id } = contributor
-      contributor.pages = collections.allSorted.flatMap(
-        (page) => {
-          const { data, url } = page
-          const { contributor, label, subtitle, title } = data
-          if (!contributor) return []
-          const includePage = Array.isArray(contributor)
-            ? contributor.find((item) => item.id === id)
-            : contributor.id === id
-          return includePage
-            ? {
-                label,
-                subtitle,
-                title,
-                url
-              }
-            : []
-        }
-      )
-      return contributor
-    }
-    const pageContributors = collections.all.flatMap(({ data }) => data.pageContributors || [])
+    // NB: filter() here removes empty items from array (!!) in tests
+    let contributors = publication.contributor.filter(c => !!c)
```

Comment:
💭 Using `filter(Boolean)` here will have the same result and may be more readable.